server/model/block.go

```diff
@@ -84,3 +116,34 @@ func (b Block) LogClone() interface{} {
 		Type:     b.Type,
 	}
 }
+
+// Patch returns an update version of the block.
+func (p *BlockPatch) Patch(block *Block) *Block {
+	if p.ParentID != nil {
+		block.ParentID = *p.ParentID
+	}
+	if p.RootID != nil {
+		block.RootID = *p.RootID
+	}
+	if p.Schema != nil {
+		block.Schema = *p.Schema
+	}
```

Comment:
nit for consistency:

```suggestion
	if p.ParentID != nil {
		block.ParentID = *p.ParentID
	}
	
	if p.RootID != nil {
		block.RootID = *p.RootID
	}
	
	if p.Schema != nil {
		block.Schema = *p.Schema
	}
```

Reply:
I don't see the difference there, what have you changed?